src/expressionDescriptor.ts

```diff
@@ -104,6 +104,12 @@ export class ExpressionDescriptor {
         this.options.monthStartIndexZero
       );
       this.expressionParts = parser.parse();
+
+      // Special handling for @reboot
+      if (this.expressionParts[0] === "@reboot") {
+        return "At system startup";
```

Comment:
I would prefer if this goes through `this.i18n`.

Reply:
Fixed in a22ef7b2bafa467daed175084a7885a1f9e72e58